### ISSUE TITLE
Installation documentation?

### ISSUE THREAD

**Issue by hellomrjones** [mhwlng/streamdeck-elite#71]
Can someone please ELI5 how to set this thing up? All the Buttons are just empty when I pull them onto the Streamdeck and don't seem to do anything. The description here is very complicated.

**Comment by mhwlng** [comment]
I don't have any more information for you

**Comment by hellomrjones** [comment]
Pretty stupid piece of software then. Useless

**Comment by bahl33ted** [comment]
What's the point of being rude and hostile toward the developer just because you need things "explained like your 5"? Not everybody has time to hold your hand.

**Comment by hellomrjones** [comment]
Point taken. Yet, software is just as good as its documentation. Anyway, going to find a better solution. Take care.
